config/webpack.lib.config.js

```diff
@@ -15,13 +15,15 @@ module.exports = {
     bundle: path.resolve(__dirname, '../src/lib-voyager.tsx'),
   },
   output: {
-    filename: "[name].js",
+    filename: "lib-voyager.js",
     path: path.resolve(__dirname, '../lib'),
     // Add /* filename */ comments to generated require()s in the output.
     pathinfo: true,
     // There are also additional JS chunk files if you use code splitting.
     chunkFilename: 'static/js/[name].chunk.js',
-    publicPath: '/lib/'
+    publicPath: '/lib/',
```

Comment:
Yeah i'm not sure what we want to do here. since there is also the built version of voyager the app which may want to go in `/build` and would drive deploys (of the application). For the moment that is why i used lib.